config/manifests/bases/vault-secrets-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,79 @@
MTcuMzksMCwwLDAsNi41Ni0yLjEyVjIwNy40MWg2LjI5djI1LjE1aC00LjhsLS42Mi0yLjEyQTI1LjE0LDI1LjE0LDAsMCwxLDk3LDIzMy4wN2MtNC4xOSwwLTUuNjgtMi45NC01LjY4LTcuNDNWMjA3LjQxWiIvPjxwYXRoIGQ9Ik0xMTUuODQsMjMyLjU2VjE5Ny4xNGw2LjMtLjg4djM2LjNaIi8+PHBhdGggZD0iTTE0MC45MywyMzIuMDlhMTcuMzksMTcuMzksMCwwLDEtNS41MywxYy00LjU5LDAtNi45MS0yLjE2LTYuOTEtNi42NnYtMTRoLTMuNzd2LTVoMy43N3YtNi4yNGw2LjMtLjg4djcuMTJoNi40NWwtLjQxLDVoLTZ2MTMuMTdhMiwyLDAsMCwwLDIuMjcsMi4yMiwxMi4xNywxMi4xNywwLDAsMCwzLjE1LS41MloiLz48cGF0aCBkPSJNNDUuNDYsMTg5Ljc0di00LjgzSDQxLjA1djQuODNIMzguNzlWMTc4LjE1aDIuMjZWMTgzaDQuNDF2LTQuODZoMi4yNnYxMS41OVptMTAuNTIsMEg1NC4xOWwtLjE2LS41N2E0Ljg1LDQuODUsMCwwLDEtMi41OS43NGMtMS41OSwwLTIuMjctMS0yLjI3LTIuNDksMC0xLjcuNzctMi4zNSwyLjU0LTIuMzVINTMuOHYtLjg4YzAtLjkyLS4yNy0xLjI1LTEuNjYtMS4yNWExMi4yOCwxMi4yOCwwLDAsMC0yLjQyLjI2bC0uMjctMS42YTExLjkzLDExLjkzLDAsMCwxLDMtLjRjMi43NCwwLDMuNTQuOTMsMy41NCwzWm0tMi4xOC0zLjE2SDUyLjE5Yy0uNzEsMC0uOTEuMTktLjkxLjgzcy4yLjg0Ljg3Ljg0YTMuNTgsMy41OCwwLDAsMCwxLjY1LS40M1ptNi43NSwzLjMzYTEwLjQxLDEwLjQxLDAsMCwxLTMtLjQ4bC4zLTEuNmExMCwxMCwwLDAsMCwyLjYuMzdjMSwwLDEuMTEtLjIxLDEuMTEtLjg1cy0uMTEtLjc3LTEuNTItMS4xYy0yLjEzLS40OS0yLjM4LTEtMi4zOC0yLjYycy43Ny0yLjQzLDMuMjUtMi40M2ExMS44LDExLjgsMCwwLDEsMi42MS4zbC0uMjEsMS42NmExNiwxNiwwLDAsMC0yLjQtLjI0Yy0uOTUsMC0xLjExLjIxLTEuMTEuNzIsMCwuNjcuMDYuNzIsMS4yNCwxLDIuNDMuNjIsMi42Ni45MiwyLjY2LDIuNjRTNjMuMTgsMTg5LjkxLDYwLjU1LDE4OS45MVptMTAtLjE3VjE4My45YzAtLjQ1LS4yLS42Ny0uNy0uNjdhNi4wNyw2LjA3LDAsMCwwLTIuMjcuNzF2NS44SDY1LjM5VjE3OGwyLjE5LjMzVjE4MmE3LjkyLDcuOTIsMCwwLDEsMy4xOC0uODFjMS40NSwwLDIsMSwyLDIuNDh2Ni4wNlptNC05LjU1di0yaDIuMTh2MlptMCw5LjU1di04LjM2aDIuMTh2OC4zNlptMy45My04LjNjMC0yLjA5LDEuMjctMy4zMSw0LjI0LTMuMzFhMTQsMTQsMCwwLDEsMy4yMy4zOGwtLjI1LDEuODdhMTguMzYsMTguMzYsMCwwLDAtMi45MS0uMjljLTEuNTYsMC0yLjA2LjUxLTIuMDYsMS43M3Y0LjRjMCwxLjIyLjUsMS43MywyLjA2LDEuNzNhMTguMzYsMTguMzYsMCwwLDAsMi45MS0uMjlsLjI1LDEuODhhMTQsMTQsMCwwLDEtMy4yMy4zN2MtMywwLTQuMjQtMS4yMi00LjI0LTMuMzFabTEyLjMsOC40N2MtMywwLTMuOC0xLjU4LTMuOC0zLjNWMTg0LjVjMC0xLjcyLjgxLTMuMywzLjgtMy4zczMuNzksMS41OCwzLjc5LDMuM3YyLjExQzk0LjU4LDE4OC4zMyw5My43NywxODkuOTEsOTAuNzksMTg5LjkxWm0wLTYuOTJjLTEuMTcsMC0xLjYxLjUtMS42MSwxLjQ0djIuMjVjMCwuOTUuNDQsMS40NSwxLjYxLDEuNDVzMS42MS0uNSwxLjYxLTEuNDV2LTIuMjVDOTIuNCwxODMuNDksOTIsMTgzLDkwLjc5LDE4M1ptOS45My4xNGExNi44NywxNi44NywwLDAsMC0yLjMsMS4yMnY1LjM5SDk2LjIzdi04LjM2aDEuODVsLjE0LjkyYTEwLDEwLDAsMCwxLDIuMjktMS4xWm04LjczLDMuODNjMCwxLjg1LS44NiwyLjk1LTIuODgsMi45NWExMi4zNCwxMi4zNCwwLDAsMS0yLjMzLS4yNHYzLjQybC0yLjE4LjMzdi0xMmgxLjc0bC4yMS43YTQuNzIsNC43MiwwLDAsMSwyLjc1LS44OGMxLjc2LDAsMi42OSwxLDIuNjksMi45MVptLTUuMjEuOTRhMTAuMDksMTAuMDksMCwwLDAsMS45My4yM2MuNzksMCwxLjEtLjM2LDEuMS0xLjEydi0yLjk1YzAtLjY5LS4yNy0xLjA3LTEuMDgtMS4wN2EzLjE0LDMuMTQsMCwwLDAtMiwuNzZaIi8+PHBhdGggZD0iTTI2LjI1LDI3LjgxLDg5Ljc3LDE1MS40NGw2NC0xMjMuNjNaTTc5Ljg5LDc2Ljg4SDY5Ljc3VjY2Ljc3SDc5Ljg5Wm0wLTE1LjE3SDY5Ljc3VjUxLjU5SDc5Ljg5Wk05NS4wNyw5Mi4wNkg4NVY4MS45NEg5NS4wN1ptMC0xNS4xOEg4NVY2Ni43N0g5NS4wN1ptMC0xNS4xN0g4NVY1MS41OUg5NS4wN1ptMTUuMDksMTUuMTdIMTAwVjY2Ljc3aDEwLjEyWk0xMDAsNjEuNzFWNTEuNTloMTAuMTJWNjEuNzFaIi8+PC9zdmc+
```

Comment:
This is `Vault_VerticalLogo_Black_RGB.svg` from the [brand kit](https://www.hashicorp.com/brand).
![Vault_VerticalLogo_Black_RGB](https://github.com/hashicorp/vault-secrets-operator/assets/444265/422aed73-506a-4e5e-904e-75132648fbf5)

Reply:
Cute